stretch/virtualbox/Vagrantfile

```diff
@@ -3,16 +3,20 @@
 
 require 'fileutils'
 
-require_relative 'subutai_hooks.rb'
-require_relative 'subutai_net.rb'
-require_relative 'subutai_config.rb'
+unless Vagrant.has_plugin?('vagrant-subutai')
+  raise 'Vagrant Subutai Cli plugin required: vagrant plugin install vagrant-subutai'
```

Comment:
Please remove the `Cli` in the plugin name everywhere, it is the Vagrant Subutai Plugin.

Reply:
Might want to keep both plugin requirements together in one place close to each other. The vbguest one is far away. Keep them close to each other please.

Reply:
got it